### PR TITLE
fix spinner

### PR DESCRIPTION
**Pull request by hchiam**
It'll prolly be a small merge conflict, but just wanted to let you see how it's intended to be used. Feel free to merge.

### CODE REVIEW

File: src/components/Main/Schedule/Home.tsx [hunk 1]
```diff
@@ -31,7 +30,7 @@ export const Home = () => {
     },
   );
 
-  if (status === 'loading') return <div>loading...</div>; // loading state
+  showLoadingSpinner(isLoading);
```
Comment (hchiam) [inline src/components/Main/Schedule/Home.tsx:40]: make use of useQuery's isLoading.

File: src/assets/global.css [hunk 2]
```diff
@@ -45,6 +45,7 @@ body.loading.message:after {
     var(--w) calc(-1 * var(--w)) 0 var(--c), calc(-1 * var(--w)) var(--w) 0 var(--c),
     0 var(--w) 0 var(--c), 0 calc(-1 * var(--w)) 0 var(--c), var(--w) 0 0 var(--c),
     calc(-1 * var(--w)) 0 0 var(--c);
+  width: 100%;
```
Comment (hchiam) [inline src/assets/global.css:48]: so it's centred on big screens or zoomed-out views

File: src/components/Main/Schedule/ScheduleTabs.tsx [hunk 3]
```diff
@@ -12,11 +12,9 @@ import {
   tabIndicatorTheme,
 } from '../../../shared/styles/theme.js';
 import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
-import { useSpinner } from '../../../shared/styles/loading-spinner';
 
 export const ScheduleTabs = ({ tabIdx, handleChange, titles }: ScheduleTabsProps) => {
   const classes = useStyles();
-  useSpinner();
```
Comment (hchiam) [inline src/components/Main/Schedule/ScheduleTabs.tsx]: no need in this component (page already has/needs it)

File: src/assets/global.css (hunk 2 above)
Comment (CedThen) [reply src/assets/global.css:48]: is there a tool that helped you write that text-shadow?

File: src/components/Main/Schedule/Home.tsx (hunk 1 above)
Comment (CedThen) [reply src/components/Main/Schedule/Home.tsx:40]: nice! react-query being useful

File: src/assets/global.css (hunk 2 above)
Comment (hchiam) [reply src/assets/global.css:48]: not really, i remember seeing something in a CodePen linked from a newsletter from css-tricks or smashingmagazine i think.  it's simple really, it's just [8 shadows, each offset in a different direction](https://codepen.io/hchiam/pen/vYGWPQr) to give the illusion of an outline.

**Pull request merged**